### DISCUSSION TITLE
Rear Light Bar not working in Heatmap or Ambient Modes

### DISCUSSION THREAD

**Discussion by SpicySide** [seerge/g-helper#2442] [General]
I have a 2024 Strix Scar 16 G634JYR and when placed in Ambient Mode the Rear Light Bar does not light up when using Heatmap or Ambient Modes. All other light modes use the Rear Light Bar except for those two. Is this expected behavior when using these modes? I don't have Armoury Crate installed to test if the Rear Light Bar works in those modes using that software. I did a search and a post from august of last year mentioned it as an issue on the 2023 models, but I'm not sure if the issue was ever figured out.

Thanks for all your work!

**Comment by seerge** [comment]
@SpicySide hello, someone else was mentioning that as well. Please check my reply and explanation in this [thread ](https://github.com/seerge/g-helper/issues/2344#issuecomment-2057546891)https://github.com/seerge/g-helper/issues/2344#issuecomment-2057546891

**Comment by seerge** [comment]
@SpicySide you can try build that I have posted in other thread 

[GHelper.zip](https://github.com/seerge/g-helper/files/14994858/GHelper.zip)

**Reply by SpicySide** [reply]
I didn't notice any difference with this version, rear light is still off in ambient and heatmap.
